playwright-e2e/lib/widget/input.ts

```diff
@@ -28,20 +33,27 @@ export default class Input extends WidgetBase {
   /**
    * Enter a text string. If value already exists, no action taken.
    * @param {string} value
+   * @param opts
    * @returns {Promise<void>}
    */
-  async fill(value: string): Promise<void> {
+  async fill(value: string | number, opts?: { dropdownOption: string, type?: boolean }): Promise<void> {
```

Comment:
`fill(value: string | number, opts?: { dropdownOption: string, type?: boolean })`

Expand function to support selecting the desired dropdown option.